### ISSUE TITLE
Print useful debug information for GraphQLServerRequest and GraphQLServerResponse

### ISSUE THREAD

**Issue by sgohlke** [dreamit-de/graphql-server#12]
At the moment we don't have a useful debug output when working with GraphQLServerRequest (previous Request). We can only see if the object is defined or undefined. Calling `JSON.stringify(request)` with these interfaces might result in errors like
`Converting circular structure to JSON`. Maybe we can add a function to create a string containing the information about headers, url and method so we have something to work with.
Should be done after #11 is merged into main.
